tests/e2e/cucumber/steps/api.ts

```diff
@@ -102,6 +102,17 @@ Given(
   }
 )
 
+Given(
+  '{string} creates {int} folder(s) in personal space using API',
+  async function (this: World, stepUser: string, numberOfFolders: number): Promise<void> {
+    const user = this.usersEnvironment.getUser({ key: stepUser })
+    for (let i = 1; i <= numberOfFolders; i++) {
+      const folderName = 'testFolder' + i
```

Comment:
```suggestion
      const folderName = `testFolder${i}`
```
if we can do like this way instead of `+`